.travis.yml

```diff
@@ -8,6 +8,13 @@ sudo: false
 # Specifies the linux distribution to run the CI on
 dist: trusty
 
+env:
+  global:
+    - CI_NODE_TOTAL=2
```

Comment:
I haven't seen this option before. What does it do? Scale up?

Reply:
Yeah I retracted from that and just use some ENV variable magic instead